index.jsx

```diff
@@ -160,14 +96,17 @@ export default {
         if (!action) {
             return;
         }
+
         if (action.type === 'DEVICE_SELECTED') {
             const { device } = action;
-            const serialport = pickSerialPort(device.serialPorts);
+
+            const serialport = device.serialPorts.find(port => port.vcom === 0);
```

Comment:
Is it better to use the `vcom` than the index? I wonder, since [in the CM we use the index](https://github.com/NordicPlayground/pc-nrfconnect-cellularmonitor/blob/88af5e08932ffe846425e208dcab4fb317848078/src/actions/deviceActions.ts#L90) and if one is better than the other, then I would like to use the same in both cases.